wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ProfiledPIDSubsystem.java

```diff
@@ -28,9 +28,11 @@
    *
    * @param controller the ProfiledPIDController to use
    */
-  public ProfiledPIDSubsystem(ProfiledPIDController controller) {
+  public ProfiledPIDSubsystem(ProfiledPIDController controller,
+                              double initialPosition) {
```

Comment:
For consistency with C++, there should still be a controller-only constructor that calls this one with initialPosition=0.